examples/buildAll.js

```diff
@@ -13,20 +13,23 @@ var exampleDirs = fs.readdirSync(__dirname).filter((file) => {
 // Ordering is important here. `npm install` must come first.
 var cmdArgs = [
   { cmd: 'npm', args: ['install'] },
-  { cmd: 'webpack', args: ['index.js'] },
-  { cmd: 'npm', args: ['test'] }
```

Comment:
webpack still there.  separate file for testAll.js.  Otherwise, the test exception exits the script early under Node 12.x, and all examples will not be built.  Temporary until Node v4 released.